mkdocs.yml

```diff
@@ -148,9 +148,11 @@ nav:
           - Library Helpers: "dev/code_reference/lib_helpers.md"
           - Library Mapping: "dev/code_reference/lib_mapping.md"
           - Mac Address: "dev/code_reference/mac.md"
+          - NIST: "dev/code_reference/nist.md"
           - OS Version: "dev/code_reference/os_version.md"
           - Password: "dev/code_reference/password.md"
           - Ping: "dev/code_reference/ping.md"
+          - Platform Mapper: "dev/code_reference/platform_mapper.md"
```

Comment:
This shouldn't be needed, see https://github.com/networktocode/netutils/pull/425/files#diff-3982ffdfe67538340b1567734143d0a1c02d63070279cc055d1e323a4301bcc7R37 for how we do this. You don't have to generate the md files themselves, just update that dictionary I referenced and then run `python development_scripts.py`.

Reply:
So, I'm a bit confused on this.

Running the pieces provided only created the lib_mapper maps for forward and reverse user docs right?

The documents in the `dev/code_reference` that were being added here were for the `nist.py `and `platform_mapper.py` modules themselves.

Reply:
The name is confusing me, can we think of something other than platform mapper?

Reply:
I'm not married to it, this was what it was being referred to up until it was made.  "If we had some kind of platform mapper that could..."

It's an abstract class builder for classes that need unified functionality.  I really don't know what to call that in regard to an actual module name.

Do you have any recommendations that would mesh well with the existing library?